### ISSUE TITLE
Tax breakdown error on multiple tax rules

### ISSUE THREAD

**Issue by Daniel-Informax** [PrestaShop/PrestaShop#14646]
**Describe the bug**
When you create a tax rule using multiples taxes (no matter if you combine then or aply one after another), and then activate the tax breakdown on the invoice, the tax base for the taxes on the invoice pdf arent displayed right.

It is tested on Prestashop 1.7.5.2 version.

**Comment by mariem-abid** [comment]
Hi @Daniel-Informax,

Thanks to add steps to reproduce your issue in new comment.

**Comment by Daniel-Informax** [comment]
1-  Create 2 taxes
2- Create one tax rule using these taxes (one after another)
3- Assign this tax rule to a product and buy it
4- Activate the tax breakdown on invoice
5- Change the invoice.tax-tab.tpl to show allways the  base price
6- Create an order buying this product
7- Print the invoice
8- You can check the invoice tax base price is wrong

![tax_error1](https://user-images.githubusercontent.com/33348383/61271168-26445b80-a7a4-11e9-8421-f51968ef2f18.png)

**Comment by khouloudbelguith** [comment]
Hi @Daniel-Informax, 

Thanks for your report & your feedback.
Isn't similar to this issue: #9920?

Thanks!

**Comment by Daniel-Informax** [comment]
Yes, but it happens not only with the shipping, but with the products

**Comment by marionf** [comment]
@khouloudbelguith 
Could you try to reproduce it, unless it's a duplicate ?

**Comment by khouloudbelguith** [comment]
Hi @Daniel-Informax,

Sorry for the late answer.
After applying these taxes, I get these results
![image](https://user-images.githubusercontent.com/16067358/63365417-83b56480-c36f-11e9-8c99-e3f0da36d810.png)

Thanks to check & feedback.

**Comment by Daniel-Informax** [comment]
Yeah, that's it. The base price is divided by 2.

**Comment by khouloudbelguith** [comment]
@Daniel-Informax, thanks for your feedback.
I manage to reproduce the issue with PS1.7.6.0 & PS1.7.5.2.
I’ll add this to the debug roadmap so that it’s fixed. If you have already fixed it on your end or if you think you can do it, please do send us a pull request!
Thanks!
